cabby/geo/walk.py

```diff
@@ -304,7 +306,8 @@ def get_pivot_near_goal(self,
     '''
 
     near_poi_con = self.map.poi.apply(lambda x: util.get_distance_between_geometries(
-      x.geometry, end_point['centroid']) < NEAR_PIVOT_DIST, axis=1)
+      x.geometry, end_point['centroid']) < NEAR_PIVOT_DIST + 
```

Comment:
nit: for readability, move the argument beginning with `end_point...` to its own line. Or compute it on a line before the call to `get_distance_between_geometries`

Reply:
also, spacing around `2*ADD_POI_DISTANCE`

Reply:
Fixed.